packages/dotlottie-js/src/tests/dotlottie-js-node.spec.ts

```diff
@@ -821,7 +821,7 @@ describe('DotLottie', () => {
         unzippedBigMergedDotLottie[`images/${animation1?.imageAssets[0]?.fileName}`] as Uint8Array,
       ).toString('base64');
 
-      const expectedDataURL = `data:image/png;base64,${base64}`;
+      const expectedDataURL = `data:image/jpeg;base64,${base64}`;
```

Comment:
I don't see any changes in the fixtures involved in this test. Am I correct in assuming this is another bug?

Reply:
@theashraf Fixture didnt change but since the mimetype function was wrong it was detecting it as png rather than jpeg which is the actual file type